Source/Core/Core/AchievementManager.h

```diff
@@ -131,6 +135,8 @@ class AchievementManager
                                                           u32* target);
   const std::unordered_map<AchievementId, LeaderboardStatus>& GetLeaderboardsInfo() const;
   RichPresence GetRichPresence();
+  bool IsDisabled() const { return m_disabled; };
+  void DisableRestoreAchievementManager(bool disabled);
```

Comment:
This is a really confusing function name. Please just use something like `SetDisabled()`, because otherwise it's completely unclear without checking the actual function.